Python/ast_unparse.c

```diff
@@ -127,6 +131,10 @@ append_ast_unaryop(_PyUnicodeWriter *writer, expr_ty e, bool omit_parens)
     case Not: op = "not "; break;
     case UAdd: op = "+"; break;
     case USub: op = "-"; break;
+    default:
+        PyErr_SetString(PyExc_SystemError,
+                        "unknown operator inside f-string");
+        return -1;
```

Comment:
Ditto: Py_UNREACHABLE()?

Reply:
in the default ?

Reply:
yes. instead of PyErr_SetString()+return -1. But it's an open question, I don't know if if's safe. That's why I asked @ambv to take a look.

Reply:
yep, I do understand ;-)